src/native/cpython_internal.c

```diff
@@ -0,0 +1,41 @@
+// CPython internal API wrapper
+#include <Python.h>
+#define Py_BUILD_CORE
+#include <string.h>
+
+/*
+    On Python 3.13+, this symbol is not exported from shared libraries.
+    Using weak linking so module loads even if symbol isn't available.
+*/
+#ifdef _WIN32
+__declspec(selectany)
+const char* (*_Py_DumpTracebackThreads)(int fd, PyInterpreterState* interp, PyThreadState* current_tstate) = NULL;
+#else
+extern const char*
+_Py_DumpTracebackThreads(int fd, PyInterpreterState* interp, PyThreadState* current_tstate) __attribute__((weak));
+#endif
+
+const char*
+crashtracker_dump_traceback_threads(int fd, PyInterpreterState* interp, PyThreadState* current_tstate)
```

Comment:
As we discuss offline, for first version, we can just simply try our best to call `_Py_DumpTracebackThreads` API and not upload runtime stack if this is not available. 

Also, for Python versions < 3.13, this is exported as a global so we can simply call it from Rust side as c-ffi without wrapping it here.